client/src/components/Button/BackButton.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
We have Back Button in many places. I figure that in these kind of cases it makes sense to define a generic Button that can be reuse in multiple spots. 

I've put it under the `components/Button` directory, which is a bit unusual compared to the rest of our components, but I think it makes sense in this scenario, Button and BackButton are closely related.